### PR TITLE
Fix typo and add instructions for changing a single dependency

### PR DESCRIPTION
**Pull request by ohemorange**

### CODE REVIEW

File: letsencrypt-auto-source/rebuild_dependencies.py [hunk 1]
```diff
@@ -217,7 +217,13 @@ def _write_requirements(dest_file, requirements, conflicts):
 # To generate this, do (with docker and package hashin installed):
 # ```
 # letsencrypt-auto-source/rebuild_dependencies.py \\
-#   letsencrypt-auto-sources/pieces/dependency-requirements.txt
+#   letsencrypt-auto-source/pieces/dependency-requirements.txt
+# ```
+# If you want to update a single dependency, run commands similar to these:
+# ```
+# pip install --no-cache-dir -e acme -e . -e certbot-apache -e certbot-nginx
```
Comment (adferrand) [inline letsencrypt-auto-source/rebuild_dependencies.py]: Why is it required to install certbot ? In my opinion, it is not strictly required to update a dependency (even if strongly advised to test it for certbot, but this is a more general requirement).

Comment (ohemorange) [reply letsencrypt-auto-source/rebuild_dependencies.py]: good question! these are the original instructions that I just pasted back in. not sure why they were written like that in the first place.

Comment (adferrand) [reply letsencrypt-auto-source/rebuild_dependencies.py]: Hello @ohemorange ! So about the question here, did you get some information about why there was these instructions to install certbot first ? @bmw maybe you have an idea about an hidden requirement here ?

Comment (bmw) [reply letsencrypt-auto-source/rebuild_dependencies.py]: I'm not 100% sure, but my guess is that the idea was in order rebuild this file, you could use `pip install` to find all the dependencies of our packages and after that you could use `hashin` to get their hashes.

Comment (adferrand) [reply letsencrypt-auto-source/rebuild_dependencies.py]: OK then in this case, this is not relevant anymore to update a single requirement for corner cases. Main case is a global update respecting all the dependency tree, and this should be done using `rebuild-dependencies.py`

Comment (ohemorange) [reply letsencrypt-auto-source/rebuild_dependencies.py]: changed

**Pull request merged**